cmd/kube-controllers/main.go

```diff
@@ -121,8 +129,57 @@ func main() {
 	// If configured to do so, start an etcdv3 compaction.
 	startCompactor(ctx, config)
 
-	// Wait forever.
-	select {}
+	// Wait forever and perform healthchecks.
+	for {
```

Comment:
I think we'll want a configuration option to disable this check, just in case. 

e.g. `HEALTHENABLED=true|false` (defaulting to true)